documents/src/pages/elements/interactive-chart.md

```diff
@@ -452,31 +452,33 @@ A Seasonality chart can be created using multiple series with the same timespan.
 ```javascript
 ::import-elements::
 
-const monthCount = {};
+const displayMonth = [];
 const months = {
-  1: "January",
-  2: "February",
-  3: "March",
-  4: "April",
-  5: "May",
-  6: "June",
-  7: "July",
-  8: "August",
-  9: "September",
-  10: "October",
-  11: "November",
-  12: "December"
-}
+  1: 'January',
+  2: 'February',
+  3: 'March',
+  4: 'April',
+  5: 'May',
+  6: 'June',
+  7: 'July',
+  8: 'August',
+  9: 'September',
+  10: 'October',
+  11: 'November',
+  12: 'December'
+};
 const chart = document.getElementById('seasonality');
 chart.config = {
   options: {
     timeScale: {
-      tickMarkFormatter: (time, tickMarkType, locale) => {
-        if(!monthCount[time.month]) {
-          monthCount[time.month] = 1;
-          return months[time.month];
+      tickMarkFormatter: (date) => {
+        const month = months[date.month];
+        // This prevents the same month from being displayed multiple times.
+        if (!displayMonth.includes(month)) {
+          displayMonth.push(month);
+          return month;
         }
-        return "";
+        return '';
       }
     },
     localization: {
```

Comment:
Let's align param name with lightweight-chart doc. It should be `time` instead of `date`
https://tradingview.github.io/lightweight-charts/docs/3.8/api#timeformatterfn

Reply:
Done!